pkg/cvss/cvssv2/cvss.go

```diff
@@ -79,6 +80,13 @@ func ParseCVSSV2(vectorStr string) (*storage.CVSSV2, error) {
 			return nil, fmt.Errorf("invalid field value %q for %q", v, k)
 		}
 	}
+	vec, err := cvss2.VectorFromString(cvssV2.GetVector())
```

Comment:
I just took a look at how we use this in [Scanner](https://github.com/stackrox/scanner/blob/master/pkg/types/types.go#L185), and I noticed some differences. We will need to round it and use `BaseScore()` instead of `Score()`.

Wish I said why this change was required: https://github.com/stackrox/scanner/pull/336 I'm sure there was some kind of scores descrepency somewhere